release/release_gold.sh

```diff
@@ -5,8 +5,33 @@ source _product_info_json.sh
 source _promotion.sh
 source _publishing.sh
 
+RELEASE_PROPERTIES_FILE="release-data/release.properties"
+LIFERAY_RELEASE_FORK_NAME=${LIFERAY_RELEASE_FORK_NAME:="liferay/liferay-portal.ee"}
+
+function tag_release {
+	if ( ! curl --data "{
```

Comment:
Adjust curl switches with retry and max timeout pls.
In this case, the timeout value probably should be relatively small. We should consult with Zsolt about that.